common/lib/transport/connectionmanager.js

```diff
@@ -1,9 +1,8 @@
 var ConnectionManager = (function() {
-	var readCookie = (typeof(Cookie) !== 'undefined' && Cookie.read);
-	var createCookie = (typeof(Cookie) !== 'undefined' && Cookie.create);
-	var eraseCookie = (typeof(Cookie) !== 'undefined' && Cookie.erase);
-	var connectionKeyCookie = 'ably-connection-key';
-	var connectionSerialCookie = 'ably-connection-serial';
+	var getFromSession    = (typeof(SessionStorage) !== 'undefined' && SessionStorage.get);
```

Comment:
Actually on second thoughts I think it'd be simpler to leave it as-is. You still need the sessionStorageSupported checks because of node (eg because window.addEventListener isn't defined in node), so the change doesn't really let you simplify things much, sadly. https://github.com/ably/ably-js/commit/285ca827787aa7cf23c9c0b7007015ea1309a3b3